FordLabs/PeopleMover
# 222_Edit_Space_Name

## Issue
Connects #222 

## What was done
- [x] Space can now be edited from the dashboard

## How to test
1. Go to Dashboard and click the ellipisis on the space tile
2. Select "edit" in dropdown
3. Edit the space name and click save 
4. Edited space name should now appear on the dashboard tile
